src/main.rs

```diff
@@ -19,13 +21,16 @@ use std::boxed::Box;
 use std::thread;
 use rustc_serialize::json;
 use fanout::{Fanout, Message, OpCode};
+use cron::CronSchedule;
+use chrono::*;
 
 #[derive(RustcDecodable, Eq, PartialEq, Clone, Debug)]
 struct Config { // TODO: Rename fields
     teamcity: teamcity::TeamcityCredentials,
     bitbucket: bitbucket::BitbucketCredentials,
     telegram: Option<telegram::TelegramCredentials>,
     run_interval: u64,
+    cron_interval: String,
```

Comment:
Can we combine this field and `run_interval` into one single [`enum`](https://doc.rust-lang.org/stable/book/enums.html) with types `u64` and `CronSchedule` (if it is `RustcDecodable` or it's easy to implement or `#derive`; otherwise, a `String` would work too).